core/templates/dev/head/pages/topic-editor-page/questions-tab/questions-tab.directive.html

```diff
@@ -1,7 +1,8 @@
 <md-card class="oppia-page-card oppia-long-text" ng-if="!questionEditorIsShown && !activeQuestion">
   <div class="questions">
     <h3 class="form-heading"> Questions included in this topic </h3>
-    <questions-list topic-id="topic.getId()"
+    <questions-list show-skill-descriptions="topic.getId()"
```

Comment:
Two things:

(a) Sorry, I made a mistake in the naming. Since this is a boolean and not a function, let's call it skillDescriptionsAreShown and selectSkillModalIsShown instead (with corresponding hyphenated attribute names).

(b) You should not be passing a topic ID to these, that doesn't make sense. You should be passing a boolean.